ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -64,7 +64,6 @@
       .build();
 
   private static final String TOPIC_NAME = "fred";
-  private static final KsqlConfig CONFIG = new KsqlConfig(ImmutableMap.of());
```

Comment:
I'm not sure if there's a compelling reason to keep this now that I'm mocking some config behavior?

Reply:
There is not. (Good call on removing this.)